### DISCUSSION TITLE
Socket.io logo policy

### DISCUSSION THREAD

**Discussion by Totodore** [socketio/socket.io#4838] [Q&A] [answered]
Hey,

I'm rewriting the readme of [socketioxide](https://github.com/totodore/socketioxide) (the rust implementation of socket.io) and was wondering if I could include the socket.io logo in it or not.

See here : https://github.com/Totodore/socketioxide/tree/chore-prepare-v0.5#readme

Thank for you're help !

**Comment by darrachequesne** [comment, marked as answer]
Yes, absolutely :+1:
